Library/OcBootManagementLib/DefaultEntryChoice.c

```diff
@@ -12,13 +12,16 @@
   WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
 **/
 
+#include <Uefi.h>
+#include <OpenCore.h>
```

Comment:
Ugh, it is not good that libraries include that header. Do we ever do it anywhere else? I designed it to be local to Platform/OpenCore originally. Most likely the things we need from there are to be moved elsewhere if at all.

Reply:
Sorry, I didn't realise this, it was for the Bootstrap path macro. How to solve it, move the macro or take a parameter from somewhere?